meta-lxatac-bsp/recipes-core/lxatac-factory-data/files/lxatac-factory-data.service

```diff
@@ -1,10 +1,13 @@
 [Unit]
 Description=Gather and Distribute Factory Data
-Before=NetworkManager.service
+DefaultDependencies=no
+After=systemd-remount-fs.service
+Before=systemd-udevd.service
+RequiresMountsFor=/run
```

Comment:
`/run` is always mounted before any service starts, so this line is unnecessary.

Reply:
I did a quick `rg RequiresMountsFor` before adding that to check if other service files in the wild had it and there were some.
But if we do not need it that's even better.

Dropped the `RequiresMountsFor=`.